core/templates/pages/exploration-editor-page/exploration-editor-page.component.ts

```diff
@@ -419,6 +419,7 @@ angular.module('oppia').component('explorationEditorPage', {
           EditabilityService.onStartTutorial();
           ctrl.tutorialInProgress = true;
         });
+        $rootScope.$apply();
```

Comment:
Hm, should this be inside the timeout method? We want it to execute only after Line 419-420 have executed right?